File: FrameLib/src/main/java/com/wyb/iocframe/util/ToastUtil.java
package com.wyb.iocframe.util;

import android.content.Context;
import android.widget.Toast;

public class ToastUtil {
	
	
	public static void mackToastSHORT(String str,Context context){
		Toast.makeText(context, str, Toast.LENGTH_SHORT).show();
	}
	public static void mackToastLONG(String str,Context context){
		Toast.makeText(context, str, Toast.LENGTH_LONG).show();
	}

}

File: lib_base/src/main/java/com/dataqin/base/utils/ToastUtil.kt
package com.dataqin.base.utils

import android.annotation.SuppressLint
import android.content.Context
import android.widget.Toast

/**
 * author:wyb
 * 统一提示框
 */
@SuppressLint("ShowToast")
object ToastUtil {
    private var toast: Toast? = null

    @Synchronized
    @JvmStatic
    fun mackToastSHORT(str: String, context: Context) {
        if (toast == null) {
            toast = Toast.makeText(context, str, Toast.LENGTH_SHORT)
        } else {
            toast?.setText(str)
        }
        toast?.show()
    }

    @Synchronized
    @JvmStatic
    fun mackToastLONG(str: String, context: Context) {
        if (toast == null) {
            toast = Toast.makeText(context, str, Toast.LENGTH_LONG)
        } else {
            toast?.setText(str)
        }
        toast?.show()
    }

}